_i18n/en.yml

```diff
@@ -147,42 +147,50 @@ publications-tweet_feed:
 publications-post:
    author_title: By
    date_title: Posted
+services-header:
+   title: Crafted solutions
+   subtitle: Tailored to you
+services-subheader:
+   title: Always listening, constantly evolving
+   subtitle: We understand that everyone is diffrent and will tailor our approach according to the needs of your business and organisation. We will help you build well-crafted software that is valuable to your needs. We provide, <strong>teams</strong> to create valuable, in-time, and quality software, <strong>expert advise</strong> to fast-track your initiatives, and <strong>training and coaching</strong> to better equip your developers with the right skills and attitudes. 
+services-section_1:
+   title: We can help with ...
+services-section_2:
+   title: Businesses we have helped
 services-software_creation:
    title: Software creation
-   subtitle: <p><strong>We will help you build well-crafted software that is valuable to your business.</strong></p>
-           <p>Our craftsmen have honed their skills in technologies and practices so that they deliver with quality on time. we can engage with you according to your needs.</p>
+   subtitle: <p><strong>We will help build well-crafted software that is valuable to your business.</strong></p>
+           <p>Our craftsmen have honed their skills in technologies and practices so that they deliver with quality on time. we will adapt our practices according to your needs.</p>
 services-software_creation-embedded_craftsmen:
    title: Embedded Craftsmen
    subtitle: Our craftsmen can enhance your teams to provide expertise in specific technologies. They can also form the core of your teams to help you instil XP practices and Software Craftsmanship values from the outset.
 services-software_creation-self_contained_teams:
    title: Self-contained Teams
-   subtitle: <p>Our craftsmen are equipped to cover all the roles required for a fully functioning Agile teams. We have experts in; UX and UI design, Front End and Back End Development, DevOps and Testing. All our craftsmen maintain an innate understanding of business analysis and work with the product owner to ensure that we constantly deliver value.</p>
-           <p>We will provide fully functional teams to help you get started quick and delivery you project in the most effective manner.</p>
+   subtitle: We will provide fully functional teams to help you deliver your project in the most effective and timely manner. Our craftsmen cover all the roles required for an Agile software developement team. We have experts in; UX and UI design, Mobile, Front End and Back End Development, DevOps, and Testing. 
```

Comment:
We will provide fully functional teams to help you deliver your project in the most effective and timely manner. Our craftsmen cover all the roles required for an agile software development team. We have experts in UX and UI design, Mobile, Front End and Back End Development, DevOps culture and Testing.